conf/messages.en

```diff
@@ -227,7 +227,7 @@ pensionSchemeMembers.description1 = You can enter the details manually, or uploa
 pensionSchemeMembers.legend = How do you want to add member details?
 pensionSchemeMembers.manualEntry = Manually enter the details
 pensionSchemeMembers.upload = Upload a file
-pensionSchemeMembers.upload.hint = Uploading a file will overwrite any details you have previously added about scheme members
+pensionSchemeMembers.upload.hint = This will overwrite any member details and payments for them you have previously added
```

Comment:
Are these content changes coming from the content document?

Reply:
yes, this is the document 
https://docs.google.com/document/d/1JG3gz9-h7K96DebUpvN3PzcM1jk8hkcSErzoyqqeh-A/edit